tests/phpunit/integration/api/term-meta/beans-term-meta/renderNonce.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Tests for _beans_term_meta::render_nonce()
+ *
+ * @package Beans\Framework\Tests\Integration\API\Term_Meta
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Integration\API\Term_Meta;
+
+use Beans\Framework\Tests\Integration\API\Term_Meta\Includes\Beans_Term_Meta_Test_Case;
+use _Beans_Term_Meta;
+
+require_once BEANS_THEME_DIR . '/lib/api/term-meta/class-beans-term-meta.php';
+require_once dirname( __DIR__ ) . '/includes/class-beans-term-meta-test-case.php';
+
+/**
+ * Class Tests_BeanTermMeta_RenderNonce
+ *
+ * @package Beans\Framework\Tests\Integration\API\Term_Meta
+ * @group   api
+ * @group   api-term-meta
+ */
+class Tests_BeansTermMeta_RenderNonce extends Beans_Term_Meta_Test_Case {
+
+	/**
+	 * Tests _beans_term_meta::render_nonce() should output nonce HTML when called.
```

Comment:
1. Fix the class name `_Beans_Term_Meta::render_nonce()`.
2. Remove the `when called` as it's not necessary.

```
Tests Beans_Term_Meta::render_nonce() should render the nonce HTML.
```